seatunnel-connectors-v2/connector-http/connector-http-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/config/HttpConfig.java

```diff
@@ -29,6 +29,38 @@ public class HttpConfig {
     public static final boolean DEFAULT_ENABLE_MULTI_LINES = false;
     public static final Option<String> URL =
             Options.key("url").stringType().noDefaultValue().withDescription("Http request url");
+    public static final Option<Long> TOTAL_PAGE_SIZE =
+            Options.key("total_page_size")
+                    .longType()
+                    .defaultValue(0L)
+                    .withDescription("total page size");
+    public static final Option<String> JSON_VERIFY_EXPRESSION =
+            Options.key("json_verify_expression")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("json verify expression ");
+    public static final Option<String> JSON_VERIFY_VALUE =
+            Options.key("json_verify_value")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("json verify value ");
```

Comment:
1. json_verify_expression:  retrieve and validate target field values using a json_path expression
2. json_verify_value:  verify if the value of the JSON verify expression matches the expected value.
3. max_page_size:  The "pageing.max_page_size" parameter is primarily used to prevent infinite loops when there is no configuration for validation through JSONPath, ensuring that the program can complete its execution even if the termination condition is not triggered.